src/shared/components/Orgs/OrgList.tsx

```diff
@@ -1,83 +1,162 @@
+> = props => {
+  const { pageSize = 20, onOrgClick, onOrgEdit, createOrg } = props;
+  const [searchValue, setSearchValue] = React.useState<string>();
+  const [{ from, size }, setPagination] = React.useState({
+    size: pageSize,
+    from: 0,
+  });
+
+  const { loading, error, data } = useNexus<OrganizationList>(
+    nexus =>
+      nexus.Organization.list({
+        from,
+        size,
+        label: searchValue,
+        deprecated: false,
+      }),
+    [from, size, searchValue]
+  );
+
+  const orgs = (data && data._results) || [];
 
-const OrgList: React.FunctionComponent<OrgListProps> = ({
-  orgs,
-  busy = false,
-  error = false,
-  onOrgClick = (org: Organization) => {},
-  onOrgEdit = () => {},
-  paginationSettings,
-  onPaginationChange,
-}) => {
-  const [items, setItems] = React.useState(orgs);
+  const paginationSettings = {
+    from,
+    total: (data && data._total) || 0,
+    pageSize: size,
+  };
 
-  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    const filtered = orgs.filter(org =>
-      org.label.toLocaleLowerCase().includes(e.target.value.toLocaleLowerCase())
-    );
-    setItems(filtered);
+  const onPaginationChange = (pageNumber: number) => {
+    setPagination({
+      size,
+      from: size * pageNumber - size,
+    });
   };
 
-  if (error) {
-    return (
-      <Empty
-        description={
-          <span>An error happened while retrieving Organizations</span>
-        }
-      />
-    );
-  }
   return (
-    <div className="OrgList">
-      {/* Don't display search for now but to be implemented soon */}
-      {/* <Search
-        className="filter"
-        placeholder="Filter by name"
-        onChange={handleChange}
-      /> */}
-      <AnimatedList
-        itemComponent={(org, i) => (
-          <OrgItem
-            key={org.label + i}
-            {...org}
-            onClick={() => onOrgClick(org)}
-            onEdit={() => onOrgEdit(org.label)}
-          />
-        )}
-        onPaginationChange={onPaginationChange}
-        makeKey={item => item.label}
-        itemName="Organization"
-        loading={busy}
-        results={items}
-        total={(paginationSettings && paginationSettings.total) || items.length}
-        paginationSettings={
-          paginationSettings && {
-            from: paginationSettings.from,
-            total: paginationSettings.total,
-            pageSize: paginationSettings.pageSize,
-          }
-        }
+    <OrgListComponent
+      {...{
+        orgs,
+        loading,
+        error,
+        onOrgClick,
+        onOrgEdit,
+        createOrg,
+        paginationSettings,
+        onPaginationChange,
+        searchValue,
+        setSearchValue,
+      }}
+    />
+  );
+};
+
+export interface OrgListProps {
+  orgs: OrgResponseCommon[];
+  loading?: boolean;
+  error?: Error;
+  onOrgClick?(orgLabel: string): void;
+  onOrgEdit?(orgLabel: string): void;
+  createOrg?(): void;
+  paginationSettings: PaginationSettings;
```

Comment:
I would get rid of pagination and so `load more` like trello for example

Reply:
> like trello
🙉